### ISSUE TITLE
Feature request: Distance to Device

### ISSUE THREAD

**Issue by oe8bck** [owntracks/android#586]
Hi,

For several tasks I would like to "measure" the distance to friends, e.g. buoy layout in a regatta. This way I found this project and I find it really cool! But I cannot see the distance in Android App, in the iPhone App this feature is in.

Theoretically I can add it by myself, but as I did program Android some years ago, I have to setup the whole toolchain again. Maybe some of you can add this info easily in the dialog which is displayed when pressing on a contact in the map view.

KR,
Ch.

**Comment by jpmens** [comment]
What motivation, exactly, do you expect us to have in order to implement that for you?

**Comment by oe8bck** [comment]
Hi,

I have three ideas of motivation:

1.) Whenever I contributed to an Open-Source project, I appreciated the direct feedback from the users. In commercial projects, the feedback loop is muuuuch longer or even not existing, And as this project is using the "Eclipse Public License 1.0", I thought the developers thereof also like to know what can be improved.
2.) I think it's quite easily to add another item in the TID, the tracker ID status dialog. So maybe someone who is currently working on the source code can add this feature
3.) As I programmed for Android several time ago, I think it takes some time to set it up again. And I do have other tasks in my spare time, so maybe I would do it a little bit different. Writing a small (Pythoin?) program as MQTT client calculating the distance and access it via HTTP.

Thanks in advance for all your support and also for this nice ecosystem which is exactly what I thought of for my use case. Just to be clear, the usage will not be commercial, its use will be for hobby regattas - there are still people doing sports just for fun without dollar signs in their eyes ;-)

KR,
Christof
